main.tf

```diff
@@ -71,3 +74,24 @@ resource "tfe_variable" "sensitive_terraform_variables" {
   sensitive    = true
   workspace_id = tfe_workspace.default.id
 }
+
+resource "tfe_variable" "clear_text_hcl_variables" {
+  for_each = var.clear_text_hcl_variables
+
+  key          = each.key
+  value        = each.value
+  category     = "terraform"
+  hcl          = true
+  workspace_id = tfe_workspace.default.id
+}
+
+resource "tfe_variable" "sensitive_hcl_variables" {
+  for_each = var.sensitive_hcl_variables
+
+  key          = each.key
+  value        = each.value.sensitive
+  category     = "terraform"
+  hcl          = true
+  sensitive    = true
+  workspace_id = tfe_workspace.default.id
+}
```

Comment:
Please place these two resources between the `env` and `terraform` ones and make sure to end the file with a newline.